scripts/governance/expireVotes.py

```diff
@@ -47,23 +47,23 @@ def completed_votes():
 
     # 2nd fund
     fund_proposal(encoded_pid, _each_funding, addr_list[1])
```

Comment:
Why do we check the outcome instead of the state? It won't change until it's completed. Not a big deal, just a question.